views/uproot-dialog.html

```diff
@@ -107,52 +86,57 @@ <h2 class="subtitle">{{ gettext("Publish Your Remix") }}</h2>
         </div>
       </div>
 
-      <!-- begin body -->
       <div class="body">
-        <div class="publish-disable-overlay" style="display: none;">
-          <h1>{{ gettext("Sign in to publish your remix") }}</h1>
-        </div>
-
-        <div class="publish-enable-overlay" style="display: none;">
-          <div id="choose-publish-method">
-            <h1>{{ gettext("Instantly publish your remix to a URL that anyone can view.") }}</h1>
-            <div class="publish-option">
-              <div class="big-button to-internet">
-                  <span class="button primary">{{ gettext("Publish To Internet") }}</span>
-              </div>
-            </div>
-          </div>
-          <div id="to-internet" style="display: none;">
-            <p class="loading">{{ gettext("Publishing...") }}</p>
-            <div class="failure" style="display: none;">
-              <p>{{ gettext("Sorry, an error occurred. Please try again later.") }}</p>
-            </div>
-
-            <div class="finished" style="display: none;">
-              <h1>{{ gettext("Here is the URL for your remix that anyone can view.") }}</h1>
-              <div class="published-url">
-                <a target="_blank" href="http://foo/bar">http://foo/bar</a>
-              </div>
-            </div>
-          </div>
-        </div>
+        <button class="login">log in using id.wmo</button>
       </div>
-      <!-- end body -->
-
     </div>
-    <!-- end dialog -->
-
 
-    <script src="{{ personaHostname }}/include.js"></script>
-    <script src="/bower/webmaker-login-ux/dist/min/webmakerLogin.min.js"></script>
 
-    <script src="jquery.min.js"></script>
-    <script src="yepnope.1.0.2-min.js"></script>
-    <script src="publication.js"></script>
-
-    <!-- shim to allow auth-login to work -->
     <script id="require-js" data-csrf="{{ csrf }}"></script>
-    <script src="/js/auth-login.js"></script>
 
+    <script>
+      var login = document.querySelector("button.login");
+      login.addEventListener("click", function() {
+        checked = false;
+        window.open(
+          "http://localhost:1234/login/oauth/authorize?" + [
+            "client_id=goggles",
+            "response_type=token",
+            "scopes=user email",
+            "state=goggles"
+          ].join("&"),
+          null,
+          "_blank"
+        );
+      });
+
+      var checked = false;
+      document.addEventListener("focus", function() {
+        if (!checked) {
+          checked = true;
+          var authToken = localStorage["goggles-auth-token"];
```

Comment:
Actually native JS API: both `localStorage["key"]` and `localStorage.getItem("key")` do the same thing (for removals, though, you want `removeItem` rather than the more base JS `delete`)